runtime/service/src/main/java/org/apache/polaris/service/config/RawBehaviorChangesConfiguration.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.service.config;
+
+import io.smallrye.config.ConfigMapping;
+
+@ConfigMapping(prefix = "polaris.behavior-changes")
```

Comment:
Why not have this on the base interface?

Reply:
Because this bean is injected in `ResolvedBehaviorChangesConfiguration` which also implements BehaviorChangesConfiguration, so that would result in an infinite recursion 🤷‍♂️